src/main/java/uk/gov/companieshouse/api/strikeoffobjections/file/FileTransferApiClient.java

```diff
@@ -167,14 +167,18 @@ public FileTransferApiClientResponse delete(String requestId, String fileId) {
      * @param httpServletResponse The HttpServletResponse to stream the file to
      * @return FileTransferApiClientResponse containing the http status
      */
-    public FileTransferApiClientResponse download(String fileId, HttpServletResponse httpServletResponse) throws IOException {
+    public FileTransferApiClientResponse download(String fileId, HttpServletResponse httpServletResponse) {
         // TODO OBJ-200 replace this dummy response with the implementation.
         FileTransferApiClientResponse fileTransferApiClientResponse = new FileTransferApiClientResponse();
         fileTransferApiClientResponse.setFileId(fileId);
         fileTransferApiClientResponse.setHttpStatus(HttpStatus.OK);
 
         InputStream inputStream = new ByteArrayInputStream("This is a test".getBytes());
-        IOUtils.copy(inputStream, httpServletResponse.getOutputStream());
+        try {
+            IOUtils.copy(inputStream, httpServletResponse.getOutputStream());
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Whats the thinking behind catching the exception here and not throwing it to the service for the service to deal with?

Reply:
No it should't really be caught here, it was only put there as part of the stop gap required to add some byte content to the stub so I thought little of leaving it there. However, there are lasting benefits to throwing an exception in the service and then throwing a service exception as we do with other methods  - this should be part of the stubbed structure so chnaged this.